app/models/embed/purl/resource_file.rb

```diff
@@ -62,14 +62,20 @@ def thumbnail?
         Settings.resource_types_that_contain_thumbnails.include?(resource.type)
       end
 
+      # Returns array of VTT files
       def vtt
```

Comment:
let's change the method name to `vtts` so it appears as plural  or maybe `text_tracks` is better.

Reply:
sounds good. I may say "captions" or similar and am considering looking for the "role" attribute instead of or in addition to mime type.